layouts/404.html

```diff
@@ -0,0 +1,31 @@
+{{ define "main" }}
```

Comment:
I don't think we need a custom 404 page. I would prefer that we use the 404 from our base theme.